CHANGELOG.md

```diff
@@ -11,6 +11,10 @@ Recommendation: for ease of reading, use the following order:
 - Fixed
 -->
 
+## [Unreleased]
+### Fixed
+- GQL: datasets/createEmpty returns alias prefixed with account name 
```

Comment:
Based on the description, it seems that there will be an account alias (`kamu/ds` not `ds`) for ST as well, is that right?

If so, it's unexpected behavior

Reply:
in single tenant workspace (also before this PR changes), `datasets.byOwnerAndName` returns alias with `kamu` prefix, but `datasets.byId` does not. I guess we should return it only for multi-tenant workspace, otherwise, client couldn't find the type of a workspace.
1. Should we remove prefix from `datasets.byOwnerAndName` response?
2. Should this endpoint be allowed for single tenant workspace at all?
3. Should we remove `owner` part from responses in case of singe-tenant workspace?

You may check calls here: https://gist.github.com/demusdev/768e3acb2184d5fa031ed9d24e5593c0